source/test/string.ts

```diff
@@ -96,3 +103,19 @@ test('string.date', t => {
 	t.throws(() => m('foo' as any, m.string.date), 'Expected string to be a date, got `foo`');
 	t.throws(() => m('foo' as any, m.string.label('bar').date), 'Expected string `bar` to be a date, got `foo`');
 });
+
+test('string.lowercase', t => {
+	t.notThrows(() => m('foo', m.string.lowercase));
+	t.notThrows(() => m('foo123', m.string.lowercase));
+	t.notThrows(() => m('123', m.string.lowercase));
+	t.throws(() => m('FOO' as any, m.string.lowercase), 'Expected string to be lowercase, got `FOO`');
```

Comment:
Remove `as any`, also on the other `t.throws` calls :)

Reply:
All `as any` are removed :)